README.md

```diff
@@ -46,18 +46,18 @@ For examples of the kinds of bugs we're looking for, see [SECURITY.md](SECURITY.
 
 Requirement|Notes
 ---|---
-Go version | Go1.9 or higher
+Go version | Go1.10 or higher
```

Comment:
So we actually require 1.10?

Reply:
I thought so but looks like Tendermint works just fine with 1.9.3. 

tm-bench requires 1.10 since it uses new https://golang.org/doc/go1.10#math Round function.

Reply:
Should we revert?

Reply:
I think we should stick to 1.10, or even consider upgrading to 1.11 around launch time. If so, we can use golang's official xchacha implementation, instead of our own.

Reply:
i bumped it b/c 1.11 was released & figured we should stay only one version behind.